### PR TITLE
ShA/add topic category

### PR DESCRIPTION
**Pull request by AlekseiiShevchuk**

### CODE REVIEW

**Comment by AlekseiiShevchuk** [comment]
all problems solved

File: resources/assets/templates/topic/topicCreateNew.hbs [hunk 1]
```diff
@@ -15,10 +18,7 @@
     </div>
     <div class="topic-new-control">
         <div class="topic-new-notification checkbox">
-            <label>
-                <input type="checkbox" name="report_me">Report me when someone post a reply
-            </label>
-
+            <label><input type="checkbox" name="report_me">Report me when someone post a reply</label>
```
Comment (binaryvo) [inline resources/assets/templates/topic/topicCreateNew.hbs:21]: is this in use somewhere?

Comment (AlekseiiShevchuk) [reply resources/assets/templates/topic/topicCreateNew.hbs:21]: i commented unused functional

**Comment by binaryvo** [comment]
Needs up to date with next

**Comment by AlekseiiShevchuk** [comment]
Done!

**Pull request merged**